clang/runtime/dpct-rt/include/dpct/group_utils.hpp

```diff
@@ -708,10 +708,10 @@ class [[deprecated("Please use group_radix_sort instead")]] radix_sort {
 
 /// Load linear segment items into block format across threads
 /// Helper for Block Load
-enum load_algorithm {
+enum class load_algorithm {
+
   BLOCK_LOAD_DIRECT,
   BLOCK_LOAD_STRIPED,
-};
```

Comment:
This line closing brace line was removed and needs to be added back.

Reply:
Yes fixed